src/main/java/com/mazyavr/schedule/service/TaskService.java

```diff
@@ -0,0 +1,56 @@
+package com.mazyavr.schedule.service;
+
+import com.mazyavr.schedule.entity.ProjectEntity;
+import com.mazyavr.schedule.entity.TaskEntity;
+import com.mazyavr.schedule.repository.ProjectRepository;
+import com.mazyavr.schedule.repository.TaskRepository;
+import java.time.ZonedDateTime;
+import java.util.ArrayList;
+import java.util.List;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+@Service
+public class TaskService {
+
+  @Autowired
+  private TaskRepository taskRepository;
+  @Autowired
+  private ProjectRepository projectRepository;
+
+  public void delete(Long id) {
+    taskRepository.deleteById(id);
+  }
+
+  public TaskEntity add(Long projectId, String name, String description, ZonedDateTime time) {
+    TaskEntity t = new TaskEntity();
+    t.setDescription(description);
+    t.setName(name);
+    t.setTime(time);
+    t.setStatus(false);
+    ProjectEntity project = projectRepository.findById(projectId).get();
```

Comment:
Нельзя просто вызвать `get()`, а если проекта с таким айди нету? Понятно что тогда бросится исключение, спринг его поймает и просто вернёт ошибку в http ответе, но она до клиента и дойдёт в таком виде, там будет написано что-то типа `Unable to get() Optional<ProjectEntity> because it is empty` или я хрен знает. Либо даже не будет исключения а просто создастся задача не привязанная ни к какому проекту. Короче надо проверять, есть такой проект или нет, если нет бросать какое-то своё исключение со своим сообщением об ошибке